examples/folder/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.12.21"
```

Comment:
```suggestion
  required_version = ">= 1"
```

These examples don't really need to require a Terraform version in most cases, unless they depend on specific features not available in 1.0.0.

Reply:
I was trying to appease the tflint gods